Ontime.js

```diff
@@ -29,28 +44,37 @@ function millisToString(millis) {
 
   return isNegative + hour + ':' + min + ':' + sec;
 }
+/**
+ *
+ * @param {number} millis
+ * @returns {float}
+ */
+function millisToFloat(millis) {
+  if(millis){
+    var sec = millis / 1000;
+    return Math.round(sec);
```

Comment:
the "ui":"time" shows the milliseconds anyway so maybe we should just let them be
and not round?

Reply:
The idea right now is that it is very simple to set a condition that equals a time if it is rounded, otherwise the user would need to always select a different way to compare to get the trigger to work.

On the other hand allowing advanced users to match exactly using offset calculations might allow high precision sync with other modules.

Reply:
That is some good points 
is there a easy way for you to do the rounding in Chataigne ? 
if there is we should leave the full precision
if not then it sounds like it would be best to round

but for the rounding i think we need to use floor and ceil

Reply:
I've been testing to add a setting allowing higher precision to those who might want that.
This is might be fixed in Ontime since there was a change made to make sure that the clock is never more than 32ms off (issue #940).

Reply:
> never more than 32ms off

this is for now only for the system clock, so the timers can still be out by some 100ms
(for now)

.round() is problematic for several reasons especially when we get to negative numbers

I have been manually switching between floor() and ceil() but trunc() seams to solve this very nicely.